.changeset/tender-bears-confess.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
It is not necessary to run `pnpm changeset` for a PR that only adds or updates tests.  
Could you remove this file, please?

See [wiki](https://github.com/yamada-ui/yamada-ui/wiki/Tips-for-Test#frequently-asked-questions)

Reply:
fixed in [5e85eee](https://github.com/yamada-ui/yamada-ui/pull/976/commits/5e85eee5e127277d4a6652c3a0dca9068e670410)